styles/style.css

```diff
@@ -0,0 +1,185 @@
+
+:root{
+    --clr-background: rgb(208, 225, 253);
+    --clr-prinmary:  rgb(99, 144, 228);
+    --clr-gray: #263238;
+    --clr-white: white;
+    
+    --font-size: 16px;
+    --border-radius: 0.2em;
+    --main-radius:5px;
+    --main-padding:5px;
+}
+
+
+
+#Bio {
+    text-align: center;
+    font-size: 16px;
+    font-family: "Open sans", sans-serif;
+    color: black;
+    
+}
+
+/* .bio-p{
+    color: black;
+    text-align: center;
+    margin: 2em 10em 2em 10em;
+    border-style: solid;
+    border-color: rgb(68, 102, 164);
+    font-size: 20px;
+    font-family: fantasy;
+
+} */
+
+
+a:link{
+    color: black
+}
+body{
+    margin: 0;
+    font-family: "Open Sans", sans-serif;
+    font-size: var(--font-size);
+    color: var(--clr-gray);
+    background-color: var(--clr-background);
+}
+
+.about-header{
+    padding: 0 0.2em;
+    font-size: 1.5em;
+    position: fixed;
+    left: 150px;
+}
+
+
+/* a{
+    font-family: "Open Sans", sans-serif;
+    text-decoration: none;
+    font-size: 1em;
+    color: black;
+
+} */
+
+
+.nav-list {
+    background: var(--clr-prinmary);
+    margin: 0;
+    padding: 1rem 0;
+    border-radius: var(--radius);
+    list-style: none;
+    box-shadow: 10px 10px 10px var(--clr-gray200);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+.nav-list a{
+    color: white;
+    text-decoration: none;
+    font-size: 1em;
+}
+
+li {
+    display: flex;
+    font-weight: bold;
+    color: var(--clr-white);
+    text-decoration: none;
+    margin-right: 1rem; 
+    padding: 0px 30px;
+}
+
+#wrapper-main{
+    margin:2em;
+}
+
+.header{
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: -1em;
+
+}
+
+
+#wrapper-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 1.5em;
+}
+
+body {
+    margin: 0;
+    font-family: "Open Sans", sans-serif;
+    font-size: var(--font-size);
+    color: var(--clr-gray);
+    background-color: var(--clr-background);
+}
+
+@media screen and (max-width: 550px) {
+    body {
+    --font-size: 12px;
+    }
+}
+
+.container {
+    background-color: var(--clr-white);
+    border-radius: var(--border-radius);
+    border-top: 4px solid var(--clr-primary);
+    padding: 1em;
+
+}
+
+.pfooter{
+    text-align: center;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    font-size: 1em;
+}
+
+.footer {
+    font-size: 10px;
+    text-align: center;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+}
+
+h2 {
+    margin-bottom: 0.2em;
+    margin-top: 0.5em;
+}
+
+h3 {
+    margin-bottom: 0.2em;
+    margin-top: 0.5em;
+}
+
+h4 {
+    margin-bottom: 0.2em;
+    margin-top: 0.5em;
+}
+
+.container1 {
```

Comment:
Prefer more descriptive names for the classes. Try to name them accoriding to the role they are playing.